fuzz/fuzz_targets/jit_singlepass.rs

```diff
@@ -20,9 +20,25 @@ impl Config for NoImportsConfig {
         false
     }
 }
+#[derive(Default, Arbitrary)]
+struct WasmSmithModule(ConfiguredModule<NoImportsConfig>);
+impl std::fmt::Debug for WasmSmithModule {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_str(&wasmprinter::print_bytes(self.0.to_bytes()).unwrap())
+    }
+}
```

Comment:
Can we move this into a common file?

Reply:
Agreed. Unfortunately, I don't know how to make the directory structure and set things up so that fuzzers share common library code. I'm not saying it's hard, but I did look into it once before.